oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java

```diff
@@ -62,7 +62,7 @@
     private String name;
     private String displayName;
     private int index;
-    private int mtu;
+    private long mtu;
```

Comment:
It's not enough just to change the type here.   Change the line that assigns this variable:
```
this.mtu = networkInterface.getMTU();
```
That core Java method can return a negative value.  See #1424.  Use `ParseUtil.unsignedIntToLong()` to convert the Java value when assigning it to the instance field.

Reply:
Also the unsignedInt parsing in the `toString()` can be removed once the reassignment is done.